third-party/cmake/Libvterm-tbl2inc_c.cmake

```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 2.8.7)
 
-set(HEX_ALPHABET "0123456789ABCDEF")
+set(HEX_ALPHABET "0123456789abcdef")
```

Comment:
I had to make the hex numbers lowercase and pad them so the output would be completely identical.